storageprovider/src/main/java/org/duracloud/storage/provider/StorageProviderBase.java

```diff
@@ -187,6 +188,18 @@ public void setSpaceACLs(String spaceId, Map<String, AclType> spaceACLs) {
         doSetSpaceProperties(spaceId, newProps);
     }
 
+    @Override
```

Comment:
What is the purpose in adding these empty overrides in the base class? Without these overrides, implementing classes are required to provide an implementation, which is likely preferred.

Reply:
Both mistakenly added.